overlord/snapstate/storehelpers_test.go

```diff
@@ -351,3 +351,110 @@ func (s *snapmgrTestSuite) TestInstallSizeErrorNoDownloadInfo(c *C) {
 	_, err := snapstate.InstallSize(st, []snapstate.MinimalInstallInfo{snapstate.InstallSnapInfo{Info: snap1}}, 0)
 	c.Assert(err, ErrorMatches, `internal error: download info missing.*`)
 }
+
+func (s *snapmgrTestSuite) TestInstallSizeWithPrereqNoStore(c *C) {
+	st := s.state
+	st.Lock()
+	defer st.Unlock()
+
+	repo := interfaces.NewRepository()
+	ifacerepo.Replace(st, repo)
+
+	s.setupInstallSizeStore()
```

Comment:
I think that `installSizeTestStore.SnapAction(..)`  should be extended to update `f.backendOps`, without this I think that all the `c.Assert(s.fakeStore.fakeBackend.ops, HasLen, 0)` checks will not really test that we are not calling the store.

Reply:
`installSizeTestStore.SnapAction` calls `fakestore.SnapAction` which updates the backend ops and before that it would also panic since the snap names are unexpected

Reply:
Indeed, you're right. Nonetheless it's probably better to avoid `s.fakeStore.fakeBackend.ops` check if it's not doing anything (may be slightly misleading).